content/_data/changelogs/weekly.yml

```diff
@@ -13969,6 +13969,124 @@
   # pull: 6189 (PR title: Confirm that UMOMNEE2 is translated to UMOMNEE in legacy API)
   # pull: 6196 (PR title: Fix UtilTest#createDirectoriesInRoot() on mac)
 
+  - version: '2.333'
+    date: 2022-01-31
+    changes:
+      - type: TODO
+        category: developer
+        pull: 6231
+        authors:
+          - dependabot[bot]
+        pr_title: Bump XStream from 1.4.18 to 1.4.19
+        message: |-
+          Jenkins has upgraded the [XStream](https://x-stream.github.io/) library from 1.4.18 (released on August 22, 2021) to 1.4.19 (released on January 29, 2022). This maintenance release of XStream addresses the security vulnerability [CVE-2021-43859](https://x-stream.github.io/CVE-2021-43859.html), when unmarshalling highly recursive collections or maps causing a Denial of Service.
+      - type: rfe
+        category: rfe
+        pull: 6228
+        authors:
+          - basil
+        pr_title: Notify `systemd(1)` about start-up completion and other service
+          status changes
+        message: |-
+          Startup completion notification is available with <code>systemd(1)</code>.
+      - type: rfe
+        category: rfe
+        pull: 6180
+        issue: 67674
+        authors:
+          - Wadeck
+          - daniel-beck
+        pr_title: "[JENKINS-67674] Update bundled dependencies after the advisory"
+        message: |-
+          Update the bundled Mailer Plugin from 1.32.1 to 408.vd726a_1130320 based on [security advisory](https://www.jenkins.io/security/advisory/2022-01-12/).
```

Comment:
"The bundled Miler Plugin is updated from 1.32.1 to 408vd72a_ ..."

Is that really the right version number for the updated package?  Did they completely change their system of naming versions?

Reply:
Yes, that's a version number from [JEP-229 continuous delivery of plugins](https://github.com/jenkinsci/jep/blob/master/jep/229/README.adoc).